tests/unit/amo/components/TestInstallButtonWrapper.js

```diff
@@ -241,50 +264,32 @@ describe(__filename, () => {
   });
 
   it('passes an add-on to AMInstallButton', () => {
-    const addon = createInternalAddonWithLang({
-      ...fakeAddon,
-      type: ADDON_TYPE_STATIC_THEME,
-    });
+    addon = { ...fakeTheme };
```

Comment:
This could either be `addon = fakeTheme;` or `addon.type = ADDON_TYPE_STATIC_THEME`.